content/design-systems-for-developers/react/en/review.md

```diff
@@ -15,104 +15,121 @@ Previously, I wrote that design systems are a [single point of failure](https://
+![Result of our first Chromatic build](/design-systems-for-developers/chromatic-first-build.png)
 
-![Choosing our repository on Netlify](/design-systems-for-developers/netlify-choose-repository.png)
+Congratulations! Now that you set up the infrastructure to publish Storybook, let's improve it with continuous integration.
 
-Enter the `storybook-build` command for Netlify to run whenever you commit.
+## Continuous integration
 
-![Setting up our first build on Netlify](/design-systems-for-developers/netlify-setup-build.png)
+Continuous integration is the defacto way to maintain modern web apps. It allows you to script behavior like tests, analysis, and deployment whenever you push code. We’ll borrow this technique to save ourselves from repetitive manual work.
 
-All going well, you should see a successful build in Netlify:
+We'll use GitHub Actions, which is free for our modest usage. The same principles apply to other CI services as well.
 
-![Succeeded running our first build in Netlify](/design-systems-for-developers/netlify-deployed.png)
+Add a `.github` directory at the top level. Then create another directory called `workflows`.
 
-Browse your published Storybook by clicking on the link. You’ll find that your local Storybook development environment is mirrored online. This makes it easy for your team to review the real rendered UI components just as you see them locally.
+Create a file called chromatic.yml like the one below. This will allow us to script how our CI process behaves. We'll start small for now and continue to improve it as we progress:
 
-![Viewing our first build in Netlify](/design-systems-for-developers/netlify-deployed-site.png)
+```yaml
+# .github/workflows/chromatic.yml
 
-Netlify runs a build command on every commit that deploys your Storybook. You’ll find a link to it in GitHub’s PR checks (we'll see that below).
+# name of our action
+name: 'Chromatic'
+# the event that will trigger the action
+on: push
 
-Congratulations! Now that you set up the infrastructure to publish Storybook, let’s demo gathering feedback.
+# what the action will do
+jobs:
+  test:
+    # the operating system it will run on
+    runs-on: ubuntu-latest
+    # the list of steps that the action will go through
+    steps:
+      - uses: actions/checkout@v1
+      - run: yarn
+      #- run: yarn build-storybook
+      - uses: chromaui/action@v1
+        # options required to the GitHub chromatic action
+        with:
+          projectToken: project-token
+          token: ${{ secrets.GITHUB_TOKEN }}
+```
 
-While we are at it, let’s add the `storybook-static` directory to our `.gitignore` file:
+Add the change with:
 
+```bash
+git add .
 ```
-# …
-storybook-static
+
+Commit it:
+
+```bash
+git commit -m "Storybook deployment with GitHub action"
 ```
 
-And commit it.
+Finally push it to the remote repository with:
 
 ```bash
-git commit -am “ignore storybook static”
+git push origin master
 ```
 
+Success! We've improved our infrastructure.
```

Comment:
```suggestion
Success! We improved our infrastructure.
```

Reply:
Suggested wording applied per feedback. Marking it as resolved.